src/postgres/commands/createPostgresServer/steps/PostgresServerCreateStep.ts

```diff
@@ -19,7 +19,7 @@ export class PostgresServerCreateStep extends AzureWizardExecuteStep<IPostgresSe
         const locationName = nonNullProp(nonNullProp(wizardContext, 'location'), 'name');
         const rgName: string = nonNullProp(nonNullProp(wizardContext, 'resourceGroup'), 'name');
         const newServerName = nonNullProp(wizardContext, 'newServerName');
-        const user: string = nonNullProp(wizardContext, 'adminUser');
+        const user: string = nonNullProp(wizardContext, 'shortUserName');
```

Comment:
I would get rid of the `user` variable since it's only used once and use `nonNullProp(wizardContext, 'shortUserName')` directly below

Alternatively, rename "user" to "shortUserName" so that we avoid the same confusion we had before